sklearn/calibration.py

```diff
@@ -1158,17 +1185,47 @@ def plot(self, *, ax=None, name=None, ref_line=True, **kwargs):
         ref_line_label = "Perfectly calibrated"
         existing_ref_line = ref_line_label in ax.get_legend_handles_labels()[1]
         if ref_line and not existing_ref_line:
-            ax.plot([0, 1], [0, 1], "k:", label=ref_line_label)
-        self.line_ = ax.plot(self.prob_pred, self.prob_true, "s-", **line_kwargs)[0]
+            ax.plot([0, 1], [0, 1], "k--", label=ref_line_label, lw=1)
+        self.line_ = ax.plot(self.prob_pred, self.prob_true, "o-", **line_kwargs)[0]
+
+        if plot_bins:
+            for x in self.bins:
+                ax.axvline(x, lw=0.5, ls="--", color="grey", zorder=-1)
```

Comment:
I assume that we should probably store those values as well if we want to be consistent.
The same for the grid.

Reply:
> rename plot_bins to grid

Good idea! I made the change.

> store those values


Do you mean storing the variable grid ?